specs/pvs/corla.pvs

```diff
@@ -1283,8 +1348,6 @@ BEGIN
 
 % @trace RDPR-6-Jul-2017 Section 25.2.2(K)
 
-%%% \subsection{Random Selection of Ballots for Audit}
```

Comment:
Did you mean to move this up above this line?
`%%% The state-wide dashboard will allow the Secretary of State`